go/ethclient/stabletokencontractlib/handler.go

```diff
@@ -0,0 +1,59 @@
+package stabletokencontractlib
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/obscuro-playground/contracts"
+	"github.com/obscuronet/obscuro-playground/go/ethclient/txhandler"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+)
+
+const methodBytesLen = 4
+
+type stableTokenHandler struct {
+	addr *common.Address
+}
+
+func NewHandler(addr *common.Address) txhandler.ContractHandler {
+	return &stableTokenHandler{
+		addr: addr,
+	}
+}
+
+func (h *stableTokenHandler) Address() *common.Address {
```

Comment:
@otherview Just make the field public?

Reply:
Yeah, I thought of it too, but I don't want the field to be overwritten :)